src/robot.coffee

```diff
@@ -211,6 +211,13 @@ class Robot
   loadFile: (path, file) ->
     ext  = Path.extname file
     full = Path.join path, Path.basename(file, ext)
+    if  require.cache[require.resolve(full)]
```

Comment:
Should we put this conditional under `if require.extensions[ext]` below? The require isn't attempted until then, so if there's not going to be anything to invalidate if it's an extention that's not supported.